dags.yaml

```diff
@@ -232,8 +241,12 @@ bqetl_asn_aggregates:
 # DAG for exporting query data marked as public to GCS
 # queries should not be explicitly assigned to this DAG (done automatically)
 bqetl_public_data_json:
-  schedule_interval: 0 4 * * *
-  description: The DAG exports query data marked as public to GCS
+  schedule_interval: 0 5 * * *
```

Comment:
In particular, mozreggression aggregates run at 04:00. If it's undesirable to delay public data publishing to GCS, then we can move mozregression up by 30 minutes.

cc @wlach

Reply:
This information is definitely not time sensitive, so no worries with this one.